### ISSUE TITLE
Simulation and Experiment

### ISSUE THREAD

**Issue by PhilipCaoChicago** [uchicago-computation-workshop/abdullah_almaatouq#52]
This paper impose a very interesting question and first well documented micro foundation for the effect of feedback and rewiring. Then it goes to simulation, the problem is somehow we don't know what is the effect when we have both feedback and rewiring, potentially there might be interesction effects. This is in general a hard point of study social interaction.

**Comment by amaatouq** [comment]
We do have both, right? in study one we fixed the feedback and manipulated the rewiring.. in study 2 we manipulated the feedback and fixed the rewiring. So in both studies, we did have a condition where there are feedback and rewiring (i.e., both are required to have an adaptive system, not one or the other).

**Comment by PhilipCaoChicago** [comment]
Yes, you estimated the pure effect for both variable. What I have in mind is another question, is how can we design a optimal network, instead of traditional idea of simply summing the pure effects, we need to consider how those factors interact. If I understand correctly this is why you did the simulation.
